core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -171,8 +173,8 @@ object Controller {
 
     def main(args: Array[String]): Unit = {
         implicit val actorSystem = ActorSystem("controller-actor-system")
-        implicit val logger = new AkkaLogging(akka.event.Logging.getLogger(actorSystem, this))
-
+        implicit val logger = new ZipkinLogging(new AkkaLogging(akka.event.Logging.getLogger(actorSystem, this)))
```

Comment:
I see. Continuing your idea I can imagine it would be nice to do the same in other places like for the database to be able to plug in another one, for action logs to be able to plug in ElasticSearch, for load balancer to be able to come with other impl that could maybe leverage HTTP instead of Kafka, for Container Pool to be able to plug in Mesos or Kube implementations. I don't have the impression that OW is at that point right now. Are you suggesting we should start with Zipkin now ?